stories/hits.stories.ts

```diff
@@ -258,4 +258,34 @@ storiesOf('Results/Hits', module)
         insightsClient: fakeInsightsClient,
       }
     )
+  )
+  .add(
+    'with html function',
+    withHits(({ search, container, instantsearch }) => {
+      search.addWidgets([
+        instantsearch.widgets.hits({
+          container,
+          templates: {
+            // @ts-ignore
+            item(hit, _, html) {
+              return html`<img height="50" src="${hit.image}" />
+                <h2 style="font-size: 1rem">
+                  ${instantsearch.highlight({
+                    attribute: 'name',
+                    hit,
+                    jsx: true,
```

Comment:
there should be another way to differentiate between the two methods. Maybe `instantsearch.js/es/components` or something (to differentiate it from a hogan helper) 🤔

Reply:
This PR [isn't meant for review](https://github.com/algolia/instantsearch.js/pull/5028#issue-1174247571) 🙂

Reply:
We can discuss this further in an RFC, but I think the final implementation will depend on whether we want to introduce this in v4 or v5. Obviously we don't want a flag like that, and we likely don't want to make the package heavier with things some users don't need.